docs/AboutUs.md

```diff
@@ -1,6 +1,5 @@
 ---
-layout: page
-title: About Us
+About Us
```

Comment:
Maybe just leave it as it is, as layout might be required for other stuff (eg. page styling) on the backend by github pages.